tests/E2E/test/selectors/BO/employee_page.js

```diff
@@ -1,26 +1,27 @@
 module.exports = {
   Employee: {
-    new_employee_button: '#page-header-desc-employee-new_employee',
-    first_name_input: '#firstname',
-    last_name_input: '#lastname',
-    email_input: '#email',
-    password_input: '#passwd',
-    profile_select: '#id_profile',
-    save_button: '#employee_form_submit_btn',
+    new_employee_button: '#page-header-desc-configuration-add',
+    first_name_input: '#employee_firstname',
+    last_name_input: '#employee_lastname',
+    email_input: '#employee_email',
+    password_input: '#employee_password',
+    profile_select: '#employee_profile',
+    save_button: '//div[@class="card-footer"]/button[contains(text(),"Save")]',
```

Comment:
You should avoid using xpath selectors.
If you need to identify that button and there's nothing to identify it, then we could add an id or a class to it.

Reply:
Thanks for the suggestion.
It would be nice to have an id for the button.

Reply:
I replace the xpath by css selector (we will replace it by #id when we will have one)